scripts/setup_docker_volume.sh

```
@@ -123,4 +123,7 @@ for v in "${volumes[@]}"; do
 	fi
 done
 
+# Create huggingface cache directory
+mkdir -p $HOME/.cache/huggingface
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)